packages/donations-app/devServer/controller/project.js

```diff
@@ -175,9 +176,20 @@ module.exports = function (app) {
         await new ProjectHistory({
           project: savedProject,
           outcomes: req.body.outcomes,
+          validators: req.body.validators,
           changedBy: req.user._id
         }).save();
 
+        // Setting validator
+        if (AccessControl.isSuperadmin(req.user)
```

Comment:
It's sufficient to require that the user is just charity-admin, we plan to gradually open the wizard to the public.

Reply:
I thought that charity admin should not be able to set validator